src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,32 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final ManufacturerDao MANUFACTURER_DAO =
```

Comment:
```suggestion
    private static final ManufacturerDao manufacturerDao =
```
this is not a constant (no magic numbers or strings)